odl/discr/discr_mappings.py

```diff
@@ -138,15 +138,28 @@ def order(self):
         return self._order
 
 
-class RawGridCollocation(FunctionSetMapping):
+class GridCollocation(FunctionSetMapping):
 
     """Function evaluation at grid points.
 
-    This is the raw `Operator` version of the default 'restriction'
-    used by all core discretization classes.
+    Given points :math:`x_1, \dots, x_n \\in \Omega \subset \mathbb{R}^d`,
```

Comment:
I agree. We need some math intro though, and here is the second best place after the introduction to operators. I think we can move some math-heavy doc from different parts of the library there, e.g. also `ProductSpaceOperator`. Something for the coming weeks.